.rubocop_todo.yml

```diff
@@ -82,7 +82,7 @@ Style/GuardClause:
   Exclude:
     - 'lib/neo4j-core/query_clauses.rb'
 
-# Offense count: 56
+# Offense count: 65
```

Comment:
Ah, sorry, the offense count increase in fine, but the "Max" counts increasing is worrying.  We try not to go backwards with our Rubocop todo values (but anything that we set in `.rubocop.yml` is our official agreed rule / max).  If you're having trouble getting the code to fit in the metrics we can help (or increase the values if it really is too cumbersome)